### ISSUE TITLE
Does Not Work With Yarn 3

### ISSUE THREAD

**Issue by enchorb** [crisp-im/node-crisp-api#59]
I get this error when attempting to run with Yarn 3 & a NX monorepo. 

Target project does not exist: `npm:fbemitter@https://github.com/crisp-dev/emitter.git#commit=695f60594bdca0c876e5c232de57702ab3151b6f`

**Comment by baptistejamin** [comment]
Hello! :)

How are you doing today?

Can you explain with details what are you trying to achieve?

**Comment by enchorb** [comment]
Hi, we have the `crisp-api@8.1.0` dependency in our `package.json` when attempting to start any project ie. `nx run-many --target=serve --projects=backend` we get the above error. Post upgrading to Yarn 3 this is the only package out of 200+ that throws an error. Everything worked fine on Yarn 1.

**Comment by baptistejamin** [comment]
Perfect! Can you please share a screenshot of the error?

I will review that with our team.

**Comment by enchorb** [comment]
```
>  NX   Failed to process project dependencies with "nx-js-graph-plugin".

   Target project does not exist: npm:fbemitter@https://github.com/crisp-dev/emitter.git#commit=695f60594bdca0c876e5c232de57702ab3151b6f
   
   Because of the error the Nx daemon process has exited. The next Nx command is going to restart the daemon process.
   If the error persists, please run "nx reset".
```

**Comment by tilmann** [comment]
Seeing the same error 
![2023 12 20 - 09 29 - Slack - #dev - Huddle@2x](https://github.com/crisp-im/node-crisp-api/assets/7314399/46f07bd2-4cb2-4e4a-b54f-7e7c9afe8b46)

Any update on this @baptistejamin

**Comment by baptistejamin** [comment]
Hello!

It should be solved with version `8.2.3`
